pylabrobot/liquid_handling/backends/hamilton/STAR_tests.py

```diff
@@ -622,7 +622,7 @@ async def test_iswap_plate_reader(self):
       pickup_distance_from_top=8.2-3.33, get_direction=GripDirection.LEFT,
       put_direction=GripDirection.FRONT)
     self._assert_command_sent_once(
-      "C0PPid0005xs10427xd0yj3286yd0zj2063zd0gr4th2450te2450gw4go1308gb1245gt20ga0gc1",
+      "C0PPid0005xs09361xd0yj3074yd0zj2063zd0gr4th2450te2450gw4go1308gb1245gt20ga0gc1",
```

Comment:
these tests have ben updated very often. perhaps we should create new tests based on the setup we had yesterday